spec/support/schemas/payment_information.json

```diff
@@ -0,0 +1,44 @@
+{
+    "data": {
+        "attributes": {
+            "responses": [
```

Comment:
maybe `payment_information_responses` (kind of long) or `payment_information` ? 🤔FE are the client maybe they have ideas @cvalarida @markgreenburg ?

Reply:
Makes sense. Its currently `responses` since that the same exact field its getting from EVSS. But making it more explicit is fine by me... I'll wait for input from FE to what they prefer.

Reply:
:thinking: I think I'm not opinionated here. `responses` isn't super descriptive, but I can't think of anything else that's more specific. They're not `accounts` or `payment_types` or `array_of_payment_information_bundles`...okay, actually it is that last one, but that's a really awful name, so let's not. :grimacing:

I'm okay with `responses`.